UPDATENOTES.md

```diff
@@ -0,0 +1,25 @@
+# Update Notes
+
+This document containts only the most important things required to know when updating Omnifactory to the next version. Please see the changelog for a complete desciption of everything that has been changed.
```

Comment:
Noticed a typo "containts" -> "contains" here, will do a full proofreading on this document later.